### DISCUSSION TITLE
GitHub Education

### DISCUSSION THREAD

**Discussion by ArielFarhi** [community/community#133271] [GitHub Education]
### Hi everyone,

Hello. I did not get the approve yet and I wanted to ask if when I will have the approve, I will be able to use compilot for free?
Thank you/

**Comment by tszdabee** [comment]
Hi @ArielFarhi, GitHub Copilot is available for free to verified students, teachers, and open-source maintainers. Once you are approved, you should have access for free.

https://github.com/education/students
